packages/text-input/text-input.scss

```diff
@@ -301,7 +304,6 @@ $text-input-selected-error-color--inverted: transparentize($text-input-error-col
 
         transition-property: height, box-shadow;
         font-weight: normal;
-        width: 100%;
```

Comment:
Denne har eg visst sletta uten at det var meininga. Kva er ryddigste måten å rette feilen mtp. commits og changelog?

Reply:
Det ryddigste er vel å gjøre en [amend](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---amend) og så en [force](https://git-scm.com/docs/git-push#Documentation/git-push.txt---force). Det er ikke for sent før den er merget :)

Reply:
Takk :) Fiksa det no.